docs/vsphere_ci/README.md

```diff
@@ -14,7 +14,7 @@ to be executed from the bastion host or any instance that has access to the vSph
 
 ## Installing Packer
 
-In case of our vSphere environment, we need to install Packer 1.6.6 on the bastion host. The installation steps are:
+In case of our vSphere environment, we need to install Packer 1.6.6+ on the bastion host. The installation steps are:
```

Comment:
@jrvaldes we can continue [this conversation](https://github.com/openshift/windows-machine-config-operator/pull/892#discussion_r798783705) here

Reply:
My take has always been to use the version we have tested with. We don't know how the newer versions will work. What if there are breaking changes?

Reply:
@saifshaikh48, given you [used](https://github.com/openshift/windows-machine-config-operator/pull/892#discussion_r798803541) Packer 1.6.6, I suggest removing the `+` .

Reply:
Makes sense, removed.